src/ui/components/Library/Team/View/NewTestRuns/Alert.tsx

```diff
@@ -0,0 +1,20 @@
+export function Alert({ children, link }: { children: string; link?: string }) {
```

Comment:
Let's move this out of `NewTestRuns` since it's used by both the test and test runs views.

It might be helpful to narrow the component a bit by exposing an enum for the alert `reason` rather than the link and message. Not a big deal but it lets us centralize the messaging and URLs. Up to you.